### ISSUE TITLE
Sending images doesn't work

### ISSUE THREAD

**Issue by konrad11901** [gave92/fbchat-sharp#3]
Hi! I've recently started using your library in my project. Unluckily, sending images doesn't work for me.

Code (FBClient is the the same as in the example):
```csharp
using System;
using System.IO;
using System.Threading.Tasks;
using fbchat_sharp.API;

namespace bot
{
    class Program
    {
        private static readonly FBClient client = new FBClient();
        static async Task Main()
        {
            Console.WriteLine("fbchat-sharp test");
            await client.DoLogin("login", "password");
            using (FileStream stream = File.OpenRead("image.png"))
            {
                await client.sendLocalImage("image.png", stream, null, "0000", ThreadType.GROUP);
            }
        }
    }
}
```
Exception:
```
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at fbchat_sharp.API.FBchatFacebookError..ctor(String message, String fb_error_code, String fb_error_message, Int32 request_status_code) in /home/konrad11901/Projects/TEST/fbchat-sharp/API/Models.cs:line 32
   at fbchat_sharp.API.Utils.<checkRequest>d__12.MoveNext() in /home/konrad11901/Projects/TEST/fbchat-sharp/API/Utils.cs:line 122
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at fbchat_sharp.API.Client.<_postFile>d__36.MoveNext() in /home/konrad11901/Projects/TEST/fbchat-sharp/API/Client.cs:line 347
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at fbchat_sharp.API.Client.<_uploadImage>d__71.MoveNext() in /home/konrad11901/Projects/TEST/fbchat-sharp/API/Client.cs:line 1482
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at fbchat_sharp.API.Client.<sendLocalImage>d__74.MoveNext() in /home/konrad11901/Projects/TEST/fbchat-sharp/API/Client.cs:line 1565
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at bot.Program.<Main>d__1.MoveNext() in /home/konrad11901/Projects/TEST/bot/Program.cs:line 17
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at bot.Program.<Main>()
```
I don't know if it's a mistake in my code or the library just doesn't work properly. If it's my mistake, I'd be thankful for pointing it out!

I'm compiling the latest version of the library from source, I'm not using the NuGet package.

**Comment by gave92** [comment]
Hi! Thank you for your interest in this library.
Code for sending images is untested (== I'm pretty sure it does not work).
I'm currently working on it, I'll update this issue when it's resolved

**Comment by gave92** [comment]
Fixed in commit 668adbdb1926733005e7d519d25e15ba28af73b7

**Comment by konrad11901** [comment]
Awesome, thanks for the update!
